File: java-code/src/classes/Position.java
package classes;

public class Position {

  private String _title;
  private String _role;

  public Position(String title, String role) {
    this._title = title;
    this._role = role;
  }

  // SETTERS
  public void SetTitle(String newTitle) {
    this._title = newTitle;
  }
  public void SetRole(String newRole) {
    this._role = newRole;
  }

  // GETTERS
  public String GetTitle() {
    return this._title;
  }
  public String GetRole() {
    return this._role;
  }
}

File: c#-code/src/classes/Position.cs
namespace APLCAssignment3.Classes
{
    public class Position
    {
        private string _title;
        private string _role;

        public Position(string title, string role)
        {
            this._title = title;
            this._role = role;
        }

        //setters
        public void SetTitle(string newTitle)
        {
            this._title = newTitle;
        }
        public void SetRole(string newRole)
        {
            this._role = newRole;
        }

        //getters
        public string GetTitle()
        {
            return this._title;
        }
        public string GetRole()
        {
            return this._role;
        }
    }
}